doc/source/data/batch_inference.rst

```diff
@@ -1,823 +1,455 @@
-or might need custom preprocessing before being fed into your models.
-To run inference workloads effectively at scale, you need to:
-
-- manage your compute infrastructure and cloud clusters
-- parallelize data processing and utilize all your cluster resources (CPUs and GPUs)
-- efficiently transfer data between cloud storage, CPUs for preprocessing, and GPUs for model inference
-
-Here's a realistic view of batch inference for modern AI applications:
-
-.. figure:: images/batch_inference_overview.png
-
-  Evaluating a batch of input data with a model to get predictions.
-
-Why use Ray Data for batch inference?
--------------------------------------
-
-There are reasons to use Ray for batch inference, even if your current
-use case does not require scaling yet:
-
-1. **Faster and Cheaper for modern Deep Learning Applications**:
-    Ray is built for
-    complex workloads and supports loading and preprocessing data with CPUs and model inference on GPUs.
-2. **Cloud, framework, and data format agnostic**:
-    Ray Data works on any cloud provider or
-    any ML framework (like PyTorch and Tensorflow) and does not require a particular file format.
-3. **Out of the box scaling**:
-    The same code that works on one machine also runs on a
-    large cluster without any changes.
-4. **Python first**:
-    You can express your inference job directly in Python instead of
-    YAML files or other formats.
-
-.. _batch_inference_quickstart:
-
-Quick Start
------------
-
-If you're impatient and want to see a copy-paste example right away,
-here are a few simple examples.
-Just pick one of the frameworks you like and run the code in your terminal.
-If you want a more detailed rundown of the same examples, skip ahead to the
-:ref:`following batch inference walk-through with Ray<batch_inference_walk_through>`.
-
-
-.. tabs::
-
-    .. group-tab:: HuggingFace
-
-        .. literalinclude:: ./doc_code/hf_quick_start.py
-            :language: python
-            :start-after: __hf_super_quick_start__
-            :end-before: __hf_super_quick_end__
-
-    .. group-tab:: PyTorch
 
-        .. literalinclude:: ./doc_code/pytorch_quick_start.py
-            :language: python
-            :start-after: __pt_super_quick_start__
-            :end-before: __pt_super_quick_end__
+Why choose Ray Data for offline inference?
+------------------------------------------
 
-    .. group-tab:: TensorFlow
+.. dropdown:: Faster and cheaper for modern deep learning applications
 
-        .. literalinclude:: ./doc_code/tf_quick_start.py
-            :language: python
-            :start-after: __tf_super_quick_start__
-            :end-before: __tf_super_quick_end__
+    Ray Data is for deep learning applications that involve both CPU preprocessing and GPU inference. Ray Data streams the working data from CPU preprocessing tasks to GPU inferencing tasks, allowing you to utilize both sets of resources concurrently.
```

Comment:
I think we want "use" in this case since it's not referring to a quantity?

> Use with caution. Don't use utilize when you mean use. It's OK to use utilize or utilization when referring to the quantity of a resource being used.

```suggestion
    Ray Data is for deep learning applications that involve both CPU preprocessing and GPU inference. Ray Data streams the working data from CPU preprocessing tasks to GPU inferencing tasks, allowing you to use both sets of resources concurrently.
```

Reply:
This is referring to actual CPU and GPU utilization